include/lsst/sphgeom/Pixelization.h

```diff
@@ -23,6 +23,11 @@
 #ifndef LSST_SPHGEOM_PIXELIZATION_H_
 #define LSST_SPHGEOM_PIXELIZATION_H_
 
+// PyInt_GetMax is not in the python 3 C API, so we define it for python >=3
+#if PY_MAJOR_VERSION >= 3
+#define PyInt_GetMax(x) 2<<31
+#endif
```

Comment:
It built on both Jenkins and my machine, so I think it is ok to keep in in this file but I can change it if that is more clear.

As for the code, PyInt_GetMax() is defined to be LONG_MAX from climits.h, which is 32 or greater (see https://docs.python.org/2/c-api/int.html and http://www.cplusplus.com/reference/climits/). I could just replace this with LONG_MAX, which should work regardless of the python version.